types/callMacros.d.ts

```diff
@@ -21,22 +21,22 @@ declare function typeOf(value: any): keyof CheckableTypes;
  * }
  * ```
  */
-declare function typeIs<T extends keyof CheckableTypes>(value: any, type: T): value is CheckableTypes[T];
+declare function $typeIs<T extends keyof CheckableTypes>(value: any, type: T): value is CheckableTypes[T];
 
 /**
  * Calls the function func with the given arguments in protected mode.
  *
  * opcall is an easier to use version of pcall. It returns a result object instead of multiple returns.
  */
-declare function opcall<T extends Array<any>, U>(
+declare function $opcall<T extends Array<any>, U>(
```

Comment:
Since TypeScript 4.6, this macro is practically useless, now that the relation between the destructured `pcall` returns is kept.
Because renaming is a breaking change, the change might as well be to remove it altogether.